### PR TITLE
Map and translations update

### PR DESCRIPTION
**Pull request by Raffox97**
**_Updates for the translations:_**
* Fixed some indenting (Italian)
* Removed unused country strings (Italian)
* Added newly used country strings (Italian)
* Edited a string that was mismatching the country code (Italian)

### CODE REVIEW

File: web/locales/it.json [hunk 1]
```diff
@@ -51,7 +55,7 @@
     },
     "tooltips": {
         "carbonintensity": "Intensità di carbonio",
-		"lowcarbon": "Basse emissioni",
+        "lowcarbon": "Basse emissioni",
```
Comment (corradio) [inline web/locales/it.json]: Is there a better translation of low carbon? Right now it seems to say low émissions

File: web/locales/en.json [hunk 2]
```diff
@@ -13,7 +13,10 @@
         "see": "see",
         "datasources": "data sources",
         "contribute": "Contribute by <a href=\"%s\" target=\"_blank\">adding your territory</a>",
-        "tip": "Tip: Click on a territory to start exploring"
+        "tip": "Tip: Click on a territory to start exploring",
+        "live": "Live",
+        "database": "Database",
```
Comment (corradio) [inline web/locales/en.json]: Actually the toolbar items shouldn't be translated as they are the name of our products.

Comment (Raffox97) [reply web/locales/en.json]: Ok, I'll rollback that edit

File: web/locales/it.json (hunk 1 above)
Comment (Raffox97) [reply web/locales/it.json]: Well, it's not the full phrase, because the full one is too long

**Comment by Raffox97** [comment]
Rollbacked all the changes regarding the toolbar

**Pull request merged**